src/main/kotlin/com/sourcegraph/cody/chat/ui/ChatPanel.kt

```diff
@@ -49,10 +58,68 @@ class ChatPanel(project: Project, chatSession: ChatSession) :
     add(lowerPanel, BorderLayout.SOUTH)
   }
 
+  fun addModelDropdown(project: Project, sessionId: String, selectedModel: ChatModel?) {
+    CodyAgentService.applyAgentOnBackgroundThread(project) { agent ->
+      val activeAccountType = CodyAuthenticationManager.instance.getActiveAccount(project)
+      //    Display available model for Enterprise user without possibility to change
+      if (activeAccountType != null && !activeAccountType.isDotcomAccount()) {
+        val chatModels = agent.server.chatModels(ChatModelsParams(sessionId))
+        chatModels.thenApplyAsync { response ->
+          ApplicationManager.getApplication().invokeLater {
+            modelDropdown.setRenderer(CodyModelComboboxRenderer())
+            add(modelDropdown, BorderLayout.NORTH)
+            response.models.forEach { modelProvider ->
+              val model = ChatModel.fromAgentName(modelProvider.model) ?: ChatModel.UNKNOWN_MODEL
```

Comment:
Compiler tells me that:
> Task :compileKotlin
w: file:///Users/pkukielka/Work/sourcegraph/jetbrains/src/main/kotlin/com/sourcegraph/cody/chat/ui/ChatPanel.kt:72:72 Elvis operator (?:) always returns the left operand of non-nullable type ChatModel
w: file:///Users/pkukielka/Work/sourcegraph/jetbrains/src/main/kotlin/com/sourcegraph/cody/chat/ui/ChatPanel.kt:92:68 Elvis operator (?:) always returns the left operand of non-nullable type ChatModel